backend/src/main/java/gov/cdc/usds/simplereport/db/repository/ApiUserRepository.java

```diff
@@ -30,4 +33,37 @@ public interface ApiUserRepository extends EternalSystemManagedEntityRepository<
 
   @Query(BASE_QUERY + " and loginEmail IN :emails" + NAME_ORDER)
   List<ApiUser> findAllByLoginEmailInOrderByName(Collection<String> emails);
+
+  String API_USER_ROLE_LEFT_JOIN = " LEFT JOIN api_user_role aur ON aur.apiUser = e";
+  String BY_ORG_AND_UNDELETED_USER = " WHERE aur.organization = :org AND e.isDeleted = false";
+  String JOINED_NAME_ORDER =
```

Comment:
nit: I think these would fit better and be easier to find declared at the top of the class with the other `NAME_ORDER` string

Reply:
Will do! Thank you!

Reply:
Done!